moveit_core/utils/src/robot_model_test_utils.cpp

```diff
@@ -120,14 +120,14 @@ void RobotModelBuilder::addChain(const std::string& section, const std::string&
     return;
   }
   // First link should already be added.
-  if (not urdf_model_->getLink(link_names[0]))
+  if (!urdf_model_->getLink(link_names[0]))
```

Comment:
``not`` this is not supported in windows?

Reply:
[not without setting the compiler to permissive mode or including a special header](https://stackoverflow.com/questions/24414124/why-does-vs-not-define-the-alternative-tokens-for-logical-operators).
Personally, I prefer the primary tokens to be used throughout the code base, over including windows-specific headers.